igvc_gazebo/nodes/control/control.cpp

```diff
@@ -1,3 +1,5 @@
+#include "control.h"
+
 #include <igvc_msgs/velocity_pair.h>
 #include <parameter_assertions/assertions.h>
 #include <ros/ros.h>
```

Comment:
Make sure to put your includes in the header file. You define `<igvc_msgs/velocity_pair.h>` and `<sensor_msgs/JointState.h>` twice.